app/src/main/java/com/example/themo/musicmarvelous/ui/main/play/PlayTrackActivity.java

```diff
@@ -0,0 +1,275 @@
+package com.example.themo.musicmarvelous.ui.main.play;
+
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.Intent;
+import android.content.ServiceConnection;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.IBinder;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.ProgressBar;
+import android.widget.SeekBar;
+import android.widget.TextView;
+
+import com.example.themo.musicmarvelous.R;
+import com.example.themo.musicmarvelous.constants.LoopMode;
+import com.example.themo.musicmarvelous.constants.ShuffleMode;
+import com.example.themo.musicmarvelous.constants.State;
+import com.example.themo.musicmarvelous.data.model.Track;
+import com.example.themo.musicmarvelous.data.repository.TrackRepository;
+import com.example.themo.musicmarvelous.service.TrackPlayerController;
+import com.example.themo.musicmarvelous.service.TrackPlayerService;
+import com.example.themo.musicmarvelous.utils.StringUtil;
+
+public class PlayTrackActivity extends AppCompatActivity implements PlayTrackContract.View,
+        View.OnClickListener, TrackPlayerController.TrackInfoListener {
+
+    private static final long DELAY_MILLIS = 500;
+    private ImageView mImvState;
+    private ImageView mImvPrevious;
+    private ImageView mImvNext;
+    private ImageView mImvLoop;
+    private ImageView mImvShuffle;
+    private TextView mTvStart;
```

Comment:
```suggestion
    private TextView mTextStart;
```
Tương tự vs các text còn lại